src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Misc.cs

```diff
@@ -108,22 +108,5 @@ internal static int ParseRawIntFile(string filePath)
 
             return ret;
         }
-
-        private static int CountOccurrences(string value, string candidate)
-        {
-            Debug.Assert(candidate != null, "CountOccurrences: Candidate string was null.");
-            int index = 0;
```

Comment:
This looks like a pre-existing bug that I'm assuming hasn't caused any issues due to the text being parsed not starting with a newline.

Reply:
@dotnet/ncl, can you confirm I'm correct here that this was a latent benign bug rather than something by design?